src/user.js

```diff
@@ -0,0 +1,42 @@
+const BaseResource = require('./base-resource')
+const UserLoan = require('./user-loan')
+const UserRequest = require('./user-request')
+
+class User extends BaseResource {
```

Comment:
I'd expect setting variables on the user instance when a new one is created from an API response.

Reply:
It's all handled in the `BaseResource` constructor. Since all the API data is put in `User.data`, `UserLoan.data` etc. each specific resource class (User, UserLoan...) doesn't have to have the code for it.

I put all the API data on the data object because otherwise some fields such as `loans` and`requests` might conflict with the instance methods on the `User` class, for example.

I think to make the interface for this simpler there needs to be some `get` and `set` methods for the `data` object